docs/device-management/1-status/README.md

```diff
@@ -0,0 +1,57 @@
+# Status
+
+Device status is described by three attributes, all of which are visible when
+viewing a device summary in the Golioth console.
+
+- **Session Established**: the time at which a device last [established a secure
+  session](/device-management/authentication) with the Golioth platform.
+- **Last Report**: the time at which a device last successfully delivered any
+  data to the Golioth platform.
+- **Setting Sync**: the time at which a device last reported the status of its
+  settings, and whether they were synchronized with the values set on the
+  Golioth platform.
+
+## Analyzing Device Status
+
+The three device status attributes can be used in concert to analyze different
+asepcts of a device's behavior. Examples for how to interpret various
+permutations of device status attributes are provided below.
+
+### Handshake Frequency
+
+It is typically optimal to maintain a session with the Golioth platform as long
```

Comment:
Eventually it would be good to expand on this more, the tradeoffs between heartbeat messages and DTLS handshakes, the potential benefits of connection ID, all depending on device behavior. But that would be for a different document and a later time :). I agree this is the right "default" advice.